.github/workflows/test_package.yml

```diff
@@ -11,8 +11,8 @@ jobs:
       # max-parallel: 6
       matrix:
         os: [ubuntu-18.04]
-        python-version: [3.5, 3.6, 3.7]
-        requires: ['minimal', 'latest']
+        python-version: [3.6, 3.7]
```

Comment:
I checked your workflow results on your fork and you could remove 3.6 and maybe add 3.8. Your numpy version seems to not be supported in 3.6.